src/bin/tikv-ctl.rs

```diff
@@ -1021,6 +1082,37 @@ fn main() {
                     ),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("remove-region")
+                .about("remove a region")
+                .arg(
+                    Arg::with_name("region")
+                        .short("r")
+                        .takes_value(true)
+                        .help("the region id"),
+                ),
+        )
+        .subcommand(
+            SubCommand::with_name("init-empty-region")
```

Comment:
May not be a good name. The command will re-initialize the region with a new region id and peers.